Sources/Fullscreen/Explore/ExploreView.swift

```diff
@@ -123,28 +147,39 @@ public final class ExploreView: UIView {
 
     // MARK: - Setup
 
-    public func configure(with sections: [ExploreSectionViewModel]) {
+    public func configure(with sections: [Section]) {
         self.sections = sections
-
-        var snapshot = NSDiffableDataSourceSnapshot<ExploreSectionViewModel, Item>()
+        self.exploreSections = []
+        self.recommendationsSection = []
+        var snapshot = NSDiffableDataSourceSnapshot<Section, Item>()
         snapshot.appendSections(sections)
 
         for section in sections {
-            switch section.layout {
-            case .hero, .squares, .twoRowsGrid:
-                let items = section.items.map {
-                    Item.regular($0, section.layout == .hero ? .big : .regular)
+            switch section {
+            case .main(let viewModel):
+                exploreSections.append(viewModel)
+                switch viewModel.layout {
+                case .hero, .squares, .twoRowsGrid:
+                    let items = viewModel.items.map {
+                        Item.regular($0, viewModel.layout == .hero ? .big : .regular)
+                    }
+                    snapshot.appendItems(items, toSection: section)
+                case .tagCloud:
+                    let items = viewModel.items.map {
+                        TagCloudCellViewModel(title: $0.title, iconUrl: $0.iconUrl)
+                    }
+                    snapshot.appendItems([Item.tagCloud(items)], toSection: section)
+                case .banner:
+                    guard let item = viewModel.items.first(where: {$0.banner != nil}), let viewModel = item.banner else { return }
+                    let banner = BrazeBannerViewModel(brazePromo: viewModel)
+                    snapshot.appendItems([Item.brazeBanner(banner)], toSection: section)
                 }
-                snapshot.appendItems(items, toSection: section)
-            case .tagCloud:
-                let items = section.items.map {
-                    TagCloudCellViewModel(title: $0.title, iconUrl: $0.iconUrl)
+            case .recommendations(let viewModels):
```

Comment:
This case's associated value is a single model and should be renamed to `viewModel`